cli/cmd/prometheus/prometheusPortForward.go

```diff
@@ -31,7 +31,7 @@ func prometheusPortForward(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("could not port-forward Prometheus: %w", err)
 	}
 
-	serviceName, err := k8s.KubeGetServiceName(root.Namespace, map[string]string{"release": root.HelmReleaseName, "app": "prometheus", "component": "server"})
+	serviceName, err := k8s.KubeGetServiceName(root.Namespace, map[string]string{"release": root.HelmReleaseName, "app": "kube-prometheus-stack-prometheus"})
```

Comment:
The labels here are hardcoded and in some other files as well. This will be fixed in upcoming refactoring.